README.md

```diff
@@ -17,51 +20,96 @@ Written in TypeScript, so type declarations are included out of the box.
- - [`@debugr/sql-formatter`] - SQL query formatter
- - [`@debugr/graphql-formatter`] - GraphQL query formatter
+Core:
+
+- [`@debugr/core`] - Boring, but necessary
+
+Plugins:
+
+- [`@debugr/apollo`] - Apollo Server request plugin
+- [`@debugr/express`] - Express request & response plugin
+- [`@debugr/insaner`] - Insaner plugin
+- [`@debugr/typeorm`] - TypeORM SQL plugin
+
+Log Handlers:
+
+- [`@debugr/console-handler`] - Log Handler to console
+- [`@debugr/elastic-handler`] - Log Handler to elastic
+- [`@debugr/html-handler`] - Log Handler to html dumps
+- [`@debugr/slack-handler`] - Log Handler to slack channel
+
+Formatter Plugins:
 
-Note that formatter plugins are installed and configured automatically with packages that
-generate data they can consume, e.g. the Express logger plugin will install and autoconfigure
-the HTTP formatter plugin.
+- [`@debugr/graphql-console-formatter`] - GraphQL query console formatter
+- [`@debugr/graphql-html-formatter`] - GraphQL query html formatter
+- [`@debugr/http-console-formatter`] - HTTP request & response console formatter
+- [`@debugr/http-html-formatter`] - HTTP request & response html formatter
+- [`@debugr/sql-console-formatter`] - SQL query console formatter
+- [`@debugr/sql-html-formatter`] - SQL query html formatter
+
+Note that formatter plugins must be installed and configured manually and with logger getter are those dependencies checked and may cause thrown of error
 
 ## Usage introduction
 
 This is an example of the raw core usage, just to show you the basics; with plugins a lot of the
 stuff will be done automatically for you.
 
 ```typescript
-import { Logger, debugr } from '@debugr/core';
-
-const debug = debugr({
-  logDir: __dirname + '/log',
-});
-
-// The Logger instance is global and can be injected wherever you need...
-const logger: Logger = debug.getLogger();
+import { 
+  Logger, 
+  Debugr, 
+  PluginManager, 
+  LogLevel,
+} from '@debugr/core';
+import { ConsoleLogHandler } from '@debugr/console-handler';
+import { HtmlLogHandler } from '@debugr/html-handler';
+
+const globalContext = {
+  applicationName: 'example',
+};
+
+const debugr = Debugr.create(globalContext);
+
+const consoleLogHandler = ConsoleLogHandler.create(
+  debugr.pluginManager, 
+  LogLevel.info,
+);
+const htmlLogHandler = HtmlLogHandler.create(
+  debugr.pluginManager, 
+  {
+    threshold: LogLevel.trace,
+    outputDir: './log',
+    gc: {
+      interval: 600,
+      threshold: LogLevel.trace,
+    },
+  },
+);
+
+debugr.registerHandler(consoleLogHandler);
+debugr.registerHandler(htmlLogHandler);
+
+// There are all dependent formatters checked and validated. In this example there are none needed, because we did not install any plugins
+const logger: Logger = debug.logger;
 
 // ... but to make Debugr aware of the execution context of your task,
 // you need to fork the logger at the beginning of the task:
 logger.fork(() => {
-    // execute your task here
-});
+  // execute your task here
```

Comment:
pořád potřebujeme mít možnost nějak "demarkovat" konkrétní job - HTTP server integrace to dělaj per-request automaticky, ale Debugr by mělo jít používat i mimo ně, takže něco jako `logger.fork()` bude pořád potřeba

Reply:
mno vsak. Asi nechapu, co chces rict. `logger.runTask` tam je

Reply:
jo aha, já se překouk, myslel jsem, že jsi smazal i to volání `logger.fork()`